docs/IP-Detection.md

```diff
@@ -25,6 +25,6 @@ Once all sources have returned an IP or timed out, the IP with most votes wins a
 
 ## Reconfiguring the Whitelisted IP
 
-While limiting the IPs able to SSH to the bastion is obviously great for security it does pose a user experience issue. Users may have their IPs change due to network conditions, restarts or simply moving location. It would be wasteful to completely reprovision a cluster every time this happens and the user would lose the progress on their current scenario.
+While limiting the IPs ability to SSH to the bastion is obviously great for security it does pose a user experience issue. Users may have their IPs change due to network conditions, restarts or simply moving location. It would be wasteful to completely reprovision a cluster every time this happens and the user would lose the progress on their current scenario.
```

Comment:
I don't think this change reads as well. We aren't limiting an IP's ability to do something. But are instead limiting "the IPs" that can do a thing.